web/ui/react-app/src/pages/targets/SeriesLabels.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Tooltip } from 'reactstrap';
+import SeriesLabel from './SeriesLabel';
+
+interface Labels {
+  [key: string]: string;
+}
+
+export interface SeriesLabelsProps {
+  discoveredLabels: Labels;
+  labels: Labels;
+}
+
+const formatLabels = (labels: Labels): string[] => Object.keys(labels).map(key => `${key}="${labels[key]}"`);
+
+const SeriesLabels: React.FC<SeriesLabelsProps> = ({ discoveredLabels, labels }: SeriesLabelsProps) => {
+  const [tooltipOpen, setTooltipOpen] = React.useState(false);
+
+  const toggle = (): void => setTooltipOpen(!tooltipOpen);
+
+  return (
+    <>
+      <div id="series-labels" className="series-labels-container">
+        {Object.keys(labels).map(key => {
+          return <SeriesLabel key={key} labelKey={key} labelValue={labels[key]} />;
+        })}
+      </div>
+      <Tooltip isOpen={tooltipOpen} target="series-labels" toggle={toggle}>
```

Comment:
The tooltip only works on the first target on the page for me, and it's flickery when it loads, I think it might be rendering all tooltips on top of each other when hovering over the first target?

Reply:
i will look into this, thanks for the heads up!

Reply:
this is fixed now :)